### PR TITLE
Issue #7750: MXNet 0.11.0 Release Feedback: README File

### PR DESCRIPTION
**Pull request by mbaijal**
Refer to Issue [#7750](https://github.com/apache/incubator-mxnet/issues/7750) -
1. Installation instructions are already included under contents in the readme
2. Updated 14 links in this PR
3. During release, we must make sure the whole R-package is removed. 
4. Already correct on master
5. Now part of this PR with the second commit

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,26 +33,26 @@ What's New
...
-* [Distributed Training](http://mxnet.io/how_to/multi_devices.html)
-* [Guide to Creating New Operators (Layers)](http://mxnet.io/how_to/new_op.html)
+* [Design Note: Design Efficient Deep Learning Data Loading Module](https://mxnet.incubator.apache.org/architecture/note_data_loading.html)
+* [MXNet on Mobile Device](https://mxnet.incubator.apache.org/how_to/smart_device.html)
+* [Distributed Training](https://mxnet.incubator.apache.org/how_to/multi_devices.html)
+* [Guide to Creating New Operators (Layers)](https://mxnet.incubator.apache.org/how_to/new_op.html)
 * [Go binding for inference](https://github.com/songtianyi/go-mxnet-predictor)
 * [Amalgamation and Go Binding for Predictors](https://github.com/jdeng/gomxnet/) - Outdated
-* [Training Deep Net on 14 Million Images on A Single Machine](http://mxnet.io/tutorials/computer_vision/imagenet_full.html)
+* [Training Deep Net on 14 Million Images on A Single Machine](http://mxnet-tqchen.readthedocs.io/en/latest/tutorials/imagenet_full.html)
```
Comment (mbaijal) [inline README.md]: The [original link](http://mxnet.io/tutorials/computer_vision/imagenet_full.html) did not work or redirect anywhere. 
On Googling for 'Training Deep Net on 14 Million Images on A Single Machine' [this is the link](http://mxnet-tqchen.readthedocs.io/en/latest/tutorials/imagenet_full.html) that came up, so I added it here.

Comment (piiswrong) [reply README.md]: No we can't point to a random site.

@madjam Is this one of the removed tutorials?

Comment (mbaijal) [reply README.md]: How do we want to handle it then? Remove the whole bullet point (Training Deep Net on 14 Million Images on A Single Machine)?

Comment (madjam) [reply README.md]: @mbaijal yep. lets not link to sites we don't control. Lets bring the content back into mxnet.
@piiswrong It happend here: https://github.com/apache/incubator-mxnet/pull/6022

**Comment by yajiedesign** [comment]
i am restart this pr ci test.

**Comment by yajiedesign** [comment]
ci seems to have a problem. Try rebase code.

**Comment by mbaijal** [comment]
ok. I will do this tomorrow and start a new PR. I am closing this for now.

**Pull request closed without merging**